net/ipv4/tcp_output.c

```diff
@@ -2683,6 +2683,16 @@ tcp_tfw_sk_write_xmit(struct sock *sk, struct sk_buff *skb,
 	tcp_set_skb_tso_segs(skb, mss_now);
 	return 0;
 }
+
```

Comment:
Please comment that the function is inspired by `tcp_write_err()`, that we can reference to the original function to get an idea about this one, and why to we send RST in it (i.e. why do we need a different function)